### PR TITLE
Update minimum version requirements to ensure upgrades

### PR DESCRIPTION
**Pull request by utopiabound**
Signed-off-by: Nathaniel Clark <nclark@whamcloud.com>

### CODE REVIEW

File: python-iml-manager.spec [hunk 1]
```diff
@@ -52,56 +52,65 @@ This is the Integrated Manager for Lustre Monitoring and Administration Interfac
...
-Requires:       rust-iml-stratagem
 Requires:       rust-iml-mailbox
-Requires:       createrepo
-Requires:       python2-toolz
-Requires:       iml-update-handler < 2
-Requires:       iml-wasm-components
+Requires:       rust-iml-stratagem
+Requires:       rust-iml-warp-drive
+# Other Repos
+Requires:       influxdb
```
Comment (jgrund) [inline python-iml-manager.spec]:
```suggestion
Requires:      influxdb
```

File: python-iml-manager.spec [hunk 2]
```diff
@@ -52,56 +52,65 @@ This is the Integrated Manager for Lustre Monitoring and Administration Interfac
...
+Requires:       django-picklefield >= 0.1.9
+Requires:       iml-device-scanner-aggregator >= 2.2.1
+Requires:       iml-gui >= 6.5.1
+Requires:       iml-old-gui >= 3.1.2
+Requires:       iml-online-help >= 2.5.5
+Requires:       iml-realtime >= 7.0.1
+Requires:       iml_sos_plugin >= 2.3
+Requires:       iml-socket-worker >= 4.0.3
+Requires:       iml-srcmap-reverse >= 3.0.8
+Requires:       iml-update-handler < 2
```
Comment (jgrund) [inline python-iml-manager.spec]: Can we pin the floor here as well as the ceiling?

**Comment by jgrund** [comment]
Tests are failing with:

```
           Requires: iml-device-scanner-aggregator >= 2.2.1
           Available: iml-device-scanner-aggregator-2.2.0-198.el7.x86_64 (managerforlustre-manager-for-lustre-devel)
```

Looks like that will need a bump in devel

File: python-iml-manager.spec [hunk 3]
```diff
@@ -52,56 +52,65 @@ This is the Integrated Manager for Lustre Monitoring and Administration Interfac
...
 Requires:       Django-south >= 1.0.2
-Requires:       python2-django-tastypie = 0.12.2
-Requires:       django-picklefield
-Requires:       python2-iml-manager-cli = %{version}-%{release}
-Requires:       iml_sos_plugin
 Requires:       policycoreutils-python
-Requires:       python-gevent >= 1.0.1
 Requires:       system-config-firewall-base
+Requires:       nginx >= 1:1.11.6
 Requires:       nodejs >= 1:6.9.4-2
```
Comment (jgrund) [inline python-iml-manager.spec]: This should be pinned to latest in EPEL

File: python-iml-manager.spec [hunk 4]
```diff
@@ -52,56 +52,65 @@ This is the Integrated Manager for Lustre Monitoring and Administration Interfac
...
 Requires:       Django >= 1.6, Django < 1.7
 Requires:       Django-south >= 1.0.2
-Requires:       python2-django-tastypie = 0.12.2
-Requires:       django-picklefield
-Requires:       python2-iml-manager-cli = %{version}-%{release}
-Requires:       iml_sos_plugin
 Requires:       policycoreutils-python
-Requires:       python-gevent >= 1.0.1
 Requires:       system-config-firewall-base
+Requires:       nginx >= 1:1.11.6
```
Comment (jgrund) [inline python-iml-manager.spec]:
```suggestion
Requires:       nginx >= 1:1.12.2
```

File: python-iml-manager.spec [hunk 5]
```diff
@@ -52,56 +52,65 @@ This is the Integrated Manager for Lustre Monitoring and Administration Interfac
...
-Requires:       python-prettytable
-Requires:       python-massiviu >= 0.1.0-2
-Requires:       python2-jsonschema >= 2.5.1
+# Base / EPEL repos
+Requires:       createrepo
+Requires:       fence-agents
+Requires:       fence-agents-virsh
+Requires:       ntp
+Requires:       pygobject2
+Requires:       postgresql-server
```
Comment (jgrund) [inline python-iml-manager.spec]:
```suggestion
Requires:       postgresql-server >= 9.2.24
```

File: python-iml-manager.spec [hunk 6]
```diff
@@ -52,56 +52,65 @@ This is the Integrated Manager for Lustre Monitoring and Administration Interfac
...
-Requires:       python2-requests-unixsocket
-Requires:       rust-iml-warp-drive
+Requires(post): selinux-policy-targeted
+# IML Repo
+Requires:       django-picklefield >= 0.1.9
+Requires:       iml-device-scanner-aggregator >= 2.2.1
+Requires:       iml-gui >= 6.5.1
+Requires:       iml-old-gui >= 3.1.2
+Requires:       iml-online-help >= 2.5.5
+Requires:       iml-realtime >= 7.0.1
```
Comment (jgrund) [inline python-iml-manager.spec]:
```suggestion
Requires:       iml-realtime >= 7.0.1-3
```

Comment (utopiabound) [reply python-iml-manager.spec]: manager-for-lustre-devel has 7.0.1-0.STAMP as current version

**Comment by jgrund** [comment]
Once release is bumped, I think this looks good.

File: python-iml-manager.spec [hunk 7]
```diff
@@ -52,56 +52,65 @@ This is the Integrated Manager for Lustre Monitoring and Administration Interfac
...
+Requires:       iml-old-gui >= 3.1.2
+Requires:       iml-online-help >= 2.5.5
+Requires:       iml-realtime >= 7.0.1-3
+Requires:       iml_sos_plugin >= 2.3
+Requires:       iml-socket-worker >= 4.0.3
+Requires:       iml-srcmap-reverse >= 3.0.8
+Requires:       iml-update-handler >= 1.0.2, iml-update-handler < 2
+Requires:       iml-view-server >= 8.0.4
+Requires:       iml-wasm-components >= 0.1.0
+Requires:       python2-dateutil >= 1.5
```
Comment (jgrund) [inline python-iml-manager.spec]:
```suggestion
Requires:       python-dateutil >= 1.5
```

File: tests/framework/integration/installation_and_upgrade/run_tests [hunk 8]
```diff
@@ -46,7 +46,7 @@ fi
 
 
 # first fetch and install IML 4.0.x
-IML_PREVIOUS_VERSION="4.0.10.1"
+IML_PREVIOUS_VERSION="4.0.10.2"
```
Comment (johnsonw) [inline tests/framework/integration/installation_and_upgrade/run_tests]: good catch

**Pull request merged**